bin/registerServerTopology.php

```diff
@@ -198,7 +198,7 @@
 
 while (!$goodIp) {
     echo $ipSelection;
-    $ipChoice = askQuestion('Which IP do you want to use as CURRENT NODE IP ?');
+    $ipChoice = askQuestion('Which IP do you want to use as CURRENT NODE IP ? ');
```

Comment:
In English we shouldn't use space before ponctuation
```suggestion
    $ipChoice = askQuestion('Which IP do you want to use as CURRENT NODE IP? ');
```